src/juju/index.js

```diff
@@ -75,7 +75,7 @@ function determineLoginParams(credentials, identityProviderAvailable) {
 
 function startPingerLoop(conn) {
   // Ping to keep the connection alive.
-  const intervalId = setInterval(() => {
+  const intervalId = window.setInterval(() => {
```

Comment:
This fixes an issue with TS types as the node.js version of `setInterval` returns a `Timer` object whereas the browser only returns an integer.